pkg/operation/botanist/component/istio/charts/istio/istio-ingress/templates/networkpolicies.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  annotations:
+    gardener.cloud/description: Disables all Egress traffic from
+      this namespace.
+  name: deny-all-egress
+  namespace: {{ .Release.Namespace }}
+spec:
+  podSelector: {}
+  policyTypes:
+  - Egress
```

Comment:
Do we need this? Isn't all traffic restricted anyways as soon as there is a `NetworkPolicy` targeting a `Pod`?

Reply:
OK, fair enough, makes sense.

Reply:
A deny all policy is best practice regarding network policies in Kubernetes. Please keep it.